src/helpers/devserver.ts

```diff
@@ -0,0 +1,87 @@
+import fetch from "cross-fetch";
```

Comment:
I'll test this with Cloudflare - I think we'll have to be able to pass the desired `fetch` implementation in.